zrpcd/zrpc_main.c

```diff
@@ -112,7 +111,7 @@ zrpc_exit (int status)
       ZRPC_FREE (tm->zrpc);
       tm->zrpc = NULL;
     }
-  
+
   cmd_terminate ();
```

Comment:
blank changes ( at least some tab and spaces already pushed) should be avoided to be changed.